mmv1/products/appengine/DomainMapping.yaml

```diff
@@ -78,7 +78,7 @@ properties:
     name: 'sslSettings'
     description: |
       SSL configuration for this domain. If unconfigured, this domain will not serve with SSL.
-    diff_suppress_func: 'sslSettingsDiffSuppress'
+    default_from_api: true
```

Comment:
Agh, I missed a step- we need to remove L50/L51 in this file. We made the file redundant and deleted it, but still try to include the file. Sorry for the extra round trip!

The lines:

```diff
- custom_code: !ruby/object:Provider::Terraform::CustomCode
-  constants: templates/terraform/constants/domain_mapping.erb
```

Reply:
No worries :) Line removed.